### DISCUSSION TITLE
Trying to add a new 2FA device results in 404

### DISCUSSION THREAD

**Discussion by dasbaumwolltier** [dani-garcia/vaultwarden#1722] [Q&A]
If I press the `Manage` button under `/#/settings/two-factor` in the UI I get this in the logs:

```
[2021-05-20 17:40:20.572][request][INFO] POST /api/two-factor/get-webauthn
[2021-05-20 17:40:20.572][response][INFO] 404 Not Found
```

I am therefore not able to add my YubiKey

**Comment by BlackDex** [comment]
Seems you are using a new web-vault then we currently support.
We do not support v2.20.x and higher right now.
Use v2.19.0 until we have updated our code to support the latest web-vault which uses webAuthn

**Comment by BlackDex** [comment]
Just as a note here. The latest code in the main branch is working with the web-vault.
